frontend/src/components/pages/events.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Container } from '@mui/material';
+import { BiArrowBack } from 'react-icons/bi';
+import { Link } from 'react-router-dom';
+import EventCard from './eventCard';
+
+const StyledArrow = styled(BiArrowBack)`
+  margin-top: 10px;
+  color: black;
+  text-align: left;
+  display: block;
+  font-size: 25px;
+`;
+
+const StyledHeader = styled.p`
+  font-family: Poppins;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 27px;
+
+  color: #000000;
+`;
+
+const StyledContainer = styled(Container)`
+  border-radius: 7px;
+  font-family: Poppins;
+  margin: 5px;
+  padding: 20px;
+  align-items: left;
+  justify-content: left;
+`;
+
+const StyledButton = styled.button`
+  background: #5f8f3e;
+  color: white;
+  font-family: Poppins;
+  font-size: 15px;
+  text-align: center;
+  border-radius: 6px;
+  border-color: #5f8f3e;
+  border-style: solid;
+  margin-top: 6px;
+  margin-bottom: 10px;
+  width: 100%;
+  height: 36px;
+  align-self: center;
+`;
+
+/*  just dummy test data for displaying events :) */
+
+const testEvents = [
+  {
+    title: "Ben's party",
+    start: new Date(121170),
+    end: new Date(),
+    location: '35.30254888400675, -120.69751392967409',
+    notes: 'very fun',
+    id: 12,
+  },
+  {
+    title: 'Kyle Concert',
+    start: new Date(22222222),
+    end: new Date(44444444444444),
+    location: 'university union',
+    notes: 'also very fun',
+    id: 13,
+  },
+  {
+    title: "Armstrong's Winter Staff Party",
+    start: new Date(),
+    end: new Date(),
+    location: 'PAC',
+    notes: "Everyone's invited!",
+    id: 99900,
+  },
+];
+
+export default function Events() {
+  const eventCards = testEvents.map((event) => {
+    return (
+      <EventCard key={event.id} title={event.title} location={event.location} />
+    );
+  });
+
+  return (
+    <div>
+      <StyledContainer maxWidth="sm">
+        <Link to="/">
+          <StyledArrow />
+        </Link>
+        <StyledHeader>Events</StyledHeader>
+        {eventCards}
+        <StyledButton>Select</StyledButton>
```

Comment:
I think we can just have users click an event to navigate to the log hours page rather than have a select. Apologies for the design volatility, please go ahead and remove this button.  

- [ ] Also optional: add the right arrow icon that was added in the designs

Reply:
I'll go ahead and switch this stuff over to the new designs now!